packages/components/src/components/link/link.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, h, Prop } from '@stencil/core';
+import { CssClassMap } from '../../utils/utils';
+import classNames from 'classnames';
+
+@Component({
+  tag: 't-link',
+  styleUrl: 'link.css',
+  shadow: true,
+})
+export class Link {
+  /** (optional) link href */
+  @Prop() public href?: string = '';
+  /** (optional) link disabled */
+  @Prop() public disabled?: boolean = false;
+  /** (optional) link variant */
+  @Prop() public underline?: boolean = false;
+  /** (optional) link open a new tag */
+  @Prop() public newTag?: boolean = false;
```

Comment:
the naming `newTag` is a bit confusing...

Reply:
rename to openNewTab